source/JsMaterialX/JsMaterialXCore/JsElement.cpp

```diff
@@ -35,11 +35,46 @@ namespace mx = MaterialX;
 
 EMSCRIPTEN_BINDINGS(element)
 {
+    ems::class_<mx::ElementEquivalenceOptions>("ElementEquivalenceOptions")
+        .constructor<>()
+        .property("performValueComparisons", &mx::ElementEquivalenceOptions::performValueComparisons)
+        .property("floatFormat", &mx::ElementEquivalenceOptions::floatFormat)
+        .property("floatPrecision", &mx::ElementEquivalenceOptions::floatPrecision)
+        .function("setAttributeExclusionList", ems::optional_override([](mx::ElementEquivalenceOptions& self, const std::vector<std::string>& exclusionList) 
+        {
+            self.attributeExclusionList = std::set<std::string>(exclusionList.begin(), exclusionList.end());
+        }));
+
     ems::class_<mx::Element>("Element")
         .smart_ptr<std::shared_ptr<mx::Element>>("Element")
         .smart_ptr<std::shared_ptr<const mx::Element>>("Element") // mx::ConstElementPtr
         .function("equals", ems::optional_override([](mx::Element& self, const mx::Element& rhs) { return self == rhs; }))
         .function("notEquals", ems::optional_override([](mx::Element& self, const mx::Element& rhs) { return self != rhs; }))
+        .function("isEquivalent", ems::optional_override([](mx::Element &self, const mx::Element& rhs, const mx::ElementEquivalenceOptions& options, 
+                                                            ems::val message) 
```

Comment:
Since the C++ interface for `isEquivalent` is still pre-release, it seems worth double-checking that we don't want it to return a simple message string instead of a vector of descriptors, matching the behavior of this JavaScript wrapper.

This wouldn't preclude us from adding the vector of descriptors further in the future, and it might be a more natural interface for users to initially work with, matching the behavior of our `validate` methods.

What are your thoughts?

Reply:
I agree it's overly complicated currently since it currently always stops after the first difference so the list degenerates to a single message. I'll look at the other PR.

Reply:
Thanks for the review, @kwokcb, and the simplification is now merged!

Reply:
@jstone-lucasfilm, I've merged the latest changes and updated the JS wrappers.